ci/playbooks/content_provider/run.yml

```diff
@@ -0,0 +1,14 @@
+---
+- hosts: all
+  gather_facts: true
+  tasks:
+    - name: Deploy content provider
+      ansible.builtin.command:
+        chdir: "{{ ansible_user_dir }}/{{ zuul.project.src_dir }}"
+        cmd: >-
+          ansible-playbook -i localhost,
```

Comment:
you may want to add `-c local` in order to NOT rely on ssh, since you're running on localhost